lib/routemap_cli.c

```diff
@@ -356,6 +356,41 @@ DEFPY_YANG(
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY_YANG(match_ipv6_next_hop_access_list, match_ipv6_next_hop_access_list_cmd,
+	   "match ipv6 next-hop access-list WORD$name",
```

Comment:
How about backward compatibility? If we use `match ipv6 next-hop ACCESSLIST6_NAME` then `match ipv6 next-hop IPV6_ADDRESS` won't work. Or do you suggest handling this logic internally (distinguishing ACL vs. IP)?

Reply:
Why won't it work? We have a lot of commands that have either IP or some sort of name in the same field – static routes, BGP neighbors, etc.

Reply:
I don't say it won't work :) If we agree with this command line _structure_ then I'll rework it here.

Reply:
You literally said "then match ipv6 next-hop IPV6_ADDRESS won't work" 😅 
I mean, we don't even need to handle it specifically inside the command – CLI already handles it well. If it's IP address, it'll match `match ipv6 next-hop X:X::X:X`, otherwise it'll match `match ipv6 next-hop WORD`.

Reply:
>Or do you suggest handling this logic internally (distinguishing ACL vs. IP)?

This is a counterpart :smile: 

Anyway, I'll fix this then.